features/breed_detail/src/main/java/com/example/catbreeds/breed_detail/BreedDetailScreen.kt

```diff
@@ -86,67 +99,88 @@ fun BreedDetailScreen(
                 modifier = Modifier
                     .fillMaxSize()
                     .padding(paddingValues)
-                    .padding(16.dp)
-                    .verticalScroll(rememberScrollState()),
-                verticalArrangement = Arrangement.spacedBy(16.dp)
+                    .verticalScroll(scrollState),
+                verticalArrangement = Arrangement.spacedBy(AppDimensions.DetailsVerticalSpacing)
             ) {
                 // Cat image
                 AsyncImage(
                     model = "https://cdn2.thecatapi.com/images/${breed.reference_image_id}.jpg",
                     contentDescription = "Image of ${breed.name}",
                     modifier = Modifier
                         .fillMaxWidth()
-                        .clip(RoundedCornerShape(8.dp)),
+                        .padding(horizontal = AppDimensions.ScreenPadding)
+                        .clip(RoundedCornerShape(AppDimensions.CardCornerRadius)),
                     contentScale = ContentScale.Crop,
-                    placeholder = painterResource(id = R.drawable.ic_menu_report_image),
-                    error = painterResource(id = R.drawable.ic_menu_close_clear_cancel)
+                    placeholder = painterResource(id = R.drawable.ic_cat_placeholder),
+                    error = painterResource(id = R.drawable.ic_cat_error)
                 )
-                // Name and origin
+
+                // Origin / Life Expectancy
                 Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    horizontalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = AppDimensions.ScreenPadding),
+                    horizontalArrangement = Arrangement.spacedBy(AppDimensions.InterItemSpacing),
```

Comment:
You can remove the `AppDimensions` for all the mentions.
```suggestion
                    horizontalArrangement = Arrangement.spacedBy(InterItemSpacing),
```

Reply:
Updated all instances to directly import only the used constants